todo_app/templates/index.html

```diff
@@ -2,17 +2,27 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+    <div class="jumbotron">
+        <h1 class="display-4">To-Do App</h1>
+        <p class="lead">Just another to-do app.</p>
+    </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Items</h2>
+            <ul class="list-group mb-4">
+              {% for item in items %}
+                <li>{{ item.title }}</li>
+              {% endfor %}
+            </ul>
+            <form id="addform" action="{{ url_for("add") }}" method="post">
+                <h2>Add a new one</h2>
+                <label for="itemname">Name</label>
+                <input type="text" id="itemname" name="name">
```

Comment:
You are calling the field `name` when there is already a pattern of calling it `title` as per line 15.

Can you change the field to be named `title`?

Reply:
Good point - will change now